beacon_node/src/lib.rs

```diff
@@ -66,6 +66,15 @@ impl<E: EthSpec> ProductionBeaconNode<E> {
         let freezer_db_path = client_config.create_freezer_db_path()?;
         let executor = context.executor.clone();
 
+        if let Some(legacy_dir) = client_config.get_existing_legacy_data_dir() {
+            warn!(
```

Comment:
This log comes up even for the default datadir (`~/.lighthouse/<network>`) which might cause unnecessary confusion. Perhaps we can skip the log for the default datadir?

Reply:
Ooh, good catch! No log when the legacy and default directory are the same!